src/app/country/country.component.ts

```diff
@@ -346,6 +353,7 @@ export class CountryComponent extends SubscriptionManager {
     };
 
     Object.values(TargetFieldName).forEach((valName: string) => {
+      const countryName = isoCountryCodesReversed[this.country];
```

Comment:
`isoCountryCodesReversed` is not very clear. what is it reversing? Is it transforming from iso to country? From country to iso? Both?
Please write a clearer name for this.

Reply:
It's the reverse mapping of the isoCountryCodes map, and the name is already clear.

What would you recommend it be called?

Reply:
I would write something that makes it clear from which direction the transformation is performing.
For example `isoCodeToCountryName` or `countryNameToIsoCode`.